### DISCUSSION TITLE
Paginação

### DISCUSSION THREAD

**Discussion by sh4rck3** [RamonSilva20/mapos#1562] [General]
Olá boa tarde, 
Sobre a paginação, apos uma consulta, a paginação iguinora o  valores passados via GET.
Ex:
http://sistema.com/index.php/os/gerenciar/10**?valores=XXXX**
os links gerados na paginação iguinoram o  campo passado via GET.
Como resolver esta questão?
Obrigado!

**Comment by Pr3d4dor** [comment]
@sh4rck3 Possivelmente é um bug, se puder criar uma issue seria o ideal.

**Comment by sh4rck3** [comment]
Certo, vou procurar uma solução! so gostaria de uma explicação, existe alguma parte do mapos relacionada a paginação ou e a padrao do codeigniter?.

**Reply by Pr3d4dor** [reply]
A paginação do mapos é feita manualmente.
